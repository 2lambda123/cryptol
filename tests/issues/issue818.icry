(+) 0b012
(+) 2z where z = 0x3
(+) 2z where z = (+) 0b012
