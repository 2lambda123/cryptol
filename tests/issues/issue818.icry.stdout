Loading module Cryptol

Parse error at issue818.icry:1:5--1:10
  malformed literal: 0b012

Parse error at issue818.icry:2:5--2:7
  malformed literal: 2z

Parse error at issue818.icry:3:5--3:7
  malformed literal: 2z
